PROJECT

```diff
@@ -10,11 +10,26 @@ repo: github.com/Azure/aks-app-routing-operator
 resources:
 - api:
     crdVersion: v1
-    namespaced: false
   controller: true
   domain: kubernetes.azure.com
   group: approuting
   kind: NginxIngressController
   path: github.com/Azure/aks-app-routing-operator/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: kubernetes.azure.com
+  group: approuting
+  kind: ExternalDNSConfiguration
```

Comment:
This name seems off. It's not an ExternalDNS configuration. It's literally ExternalDNS.

A Deployment isn't called DeploymentConfiguration, it's 1:1 with a Deployment

Reply:
this isn't 1:1 with a deployment though - it's a set of other resources as well that all configure the externaldns deployment (hence the "configuration")

Reply:
what other resources is it? It's effectively just ExternalDNS.

A Deployment isn't just a ReplicaSet. It's also a Pod. My point is that Kubernetes resources are named after what abstractions they are. This isn't a Configuration, it's literally ExternalDNS